src/components/ChannelList/hooks/listeners/useAddedToChannelNotification.js

```diff
@@ -0,0 +1,30 @@
+import { useContext, useEffect } from 'react';
+import uniqBy from 'lodash/uniqBy';
+
+import { ChatContext } from '../../../../context';
+import { getChannel } from '../../utils';
+
+export const useAddedToChannelNotification = ({
+  onAddedToChannel,
+  setChannels,
+}) => {
+  const { client } = useContext(ChatContext);
+
+  useEffect(() => {
+    const handleEvent = async (e) => {
+      if (onAddedToChannel && typeof onAddedToChannel === 'function') {
```

Comment:
made this comment in few places
```suggestion
      if (typeof onAddedToChannel === 'function') {
```